sdk/metric/pipeline.go

```diff
@@ -574,11 +598,7 @@ func newPipelines(res *resource.Resource, readers []Reader, views []View, exempl
 	return pipes
 }
 
-func (p pipelines) registerMultiCallback(c multiCallback) metric.Registration {
-	unregs := make([]func(), len(p))
-	for i, pipe := range p {
-		unregs[i] = pipe.addMultiCallback(c)
-	}
+func (p pipelines) registerMultiCallbacks(unregs []func()) metric.Registration {
```

Comment:
nit: consider removing this function, and just returning `unregisterFuncs{f: unregs}` from `RegisterCallback`

Reply:
Fixed